src/systems/implicit_system.C

```diff
@@ -43,12 +44,18 @@ ImplicitSystem::ImplicitSystem (EquationSystems & es,
 
   Parent            (es, name_in, number_in),
   matrix            (nullptr),
-  zero_out_matrix_and_rhs(true)
+  zero_out_matrix_and_rhs(true),
+  _sc               (nullptr)
 {
+  if (libMesh::on_command_line("--" + name_in + "-static-condensation"))
```

Comment:
Can we move this block to a separate method so static condensation can be enabled from code?  Or would it have to be a constructor argument?  It would seem to me that turning SC on any time before `System::init()` ought to be feasible, at least.